src/Components/EnvironmentPicker/EnvironmentPicker.tsx

```diff
@@ -405,7 +405,7 @@ const EnvironmentPicker = ({
         resources: Array<IAzureResource | string>
     ) => {
```

Comment:
Exception thrown if you delete teh value in the STorage account dropdown field. 
1. Select a value
2. Select a container
3. Clear the container
4. Clear the storage account url
5. Boom
![image](https://user-images.githubusercontent.com/57726991/186234688-c602b3d0-ef95-4471-ae11-61fe6867a91d.png)

Reply:
fixed: https://github.com/microsoft/iot-cardboard-js/pull/684/commits/8d5b111d93af7c315c728ece23ba4fbff2f310df